unix/charruad.ml

```diff
@@ -199,12 +199,8 @@ let cmd =
                          ~doc:"Log verbosity, warning|notice|debug") in
   let daemonize = Arg.(value & flag & info ["D" ; "daemon"]
                          ~doc:"Daemonize.") in
-  (* let color = *)
-  (*   let when_enum = [ "always", `Always; "never", `Never; "auto", `Auto ] in *)
-  (*   let doc = Arg.info ~docv:"WHEN" *)
-  (*       ~doc:(Printf.sprintf "Colorize the output. $(docv) must be %s." *)
-  (*               (Arg.doc_alts_enum when_enum)) ["color"] in *)
-  (*   Arg.(value & opt (enum when_enum) `Auto & doc) in *)
-  Term.(pure charruad $ configfile $ group $ pidfile $ user $ verbosity $ daemonize),
-  Term.info "charruad" ~version:"0.1" ~doc:"Charrua DHCPD"
-let () = match Term.eval cmd with `Error _ -> exit 1 | _ -> exit 0
+  Cmd.v
+    (Cmd.info "charruad" ~version:"0.1" ~doc:"Charrua DHCPD")
```

Comment:
```suggestion
    (Cmd.info "charruad" ~version:"%%VERSION%%" ~doc:"Charrua DHCPD")
```

This "watermark" of `%%VERSION%%` will be replaced by `dune subst`, either at opam-pin time, or at release time -- and it will be the complete version of charrua.(at the moment `v1.5.0`) [it is also possible to use `%%VERSION_NUM%%` which excludes the leading `v`]

Reply:
good catch, my brain didn't even read it,